velox/exec/benchmarks/PrefixSortBenchmark.cpp

```diff
@@ -0,0 +1,323 @@
+    // Copy rows to avoid sort rows already sorted.
+    std::vector<char*> sortedRows = rows;
+    PrefixSort::sort(
+        sortedRows, pool_, rowContainer, compareFlags, kDefaultSortConfig);
+  }
+
+  void runStdSort(
+      const std::vector<char*>& rows,
+      RowContainer* rowContainer,
+      const std::vector<CompareFlags>& compareFlags) {
+    std::vector<char*> sortedRows = rows;
+    std::sort(
+        sortedRows.begin(),
+        sortedRows.end(),
+        [&](const char* leftRow, const char* rightRow) {
+          for (auto i = 0; i < compareFlags.size(); ++i) {
+            if (auto result = rowContainer->compare(
+                    leftRow, rightRow, i, compareFlags[i])) {
+              return result < 0;
+            }
+          }
+          return false;
+        });
+  }
+
+  // Add benchmark manually to avoid writing a lot of BENCHMARK.
+  void addBenchmark(
+      const std::string& testName,
+      size_t numRows,
+      const RowTypePtr& rowType,
+      int iterations,
+      int numKeys,
+      bool testStdSort) {
+    auto testCase =
+        std::make_unique<TestCase>(pool_, testName, numRows, rowType, numKeys);
+    // Add benchmarks for std-sort and prefix-sort.
+    {
+      if (testStdSort) {
+        folly::addBenchmark(
+            __FILE__,
+            "StdSort_" + testCase->testName(),
+            [rows = testCase->rows(),
+             container = testCase->rowContainer(),
+             sortFlags = testCase->compareFlags(),
+             iterations = iterations,
+             this]() {
+              for (auto i = 0; i < iterations; ++i) {
+                runStdSort(rows, container, sortFlags);
+              }
+              return rows.size() * iterations;
+            });
+      }
+      folly::addBenchmark(
+          __FILE__,
+          testStdSort ? "%PrefixSort" : "PrefixSort_" + testCase->testName(),
+          [rows = testCase->rows(),
+           container = testCase->rowContainer(),
+           sortFlags = testCase->compareFlags(),
+           iterations = iterations,
+           this]() {
+            for (auto i = 0; i < iterations; ++i) {
+              runPrefixSort(rows, container, sortFlags);
+            }
+            return rows.size() * iterations;
+          });
+    }
+    testCases_.push_back(std::move(testCase));
+  }
+
+  void benchmark(
+      const std::string& prefix,
+      const std::string& keyName,
+      const std::vector<vector_size_t>& batchSizes,
+      const std::vector<RowTypePtr>& rowTypes,
+      const std::vector<int>& numKeys,
+      int32_t iterations,
+      bool testStdSort = true) {
+    for (auto batchSize : batchSizes) {
+      for (auto i = 0; i < rowTypes.size(); ++i) {
+        const auto name = fmt::format(
+            "{}_{}_{}_{}k", prefix, numKeys[i], keyName, batchSize / 1000.0);
+        addBenchmark(
+            name, batchSize, rowTypes[i], iterations, numKeys[i], testStdSort);
+      }
+    }
+  }
+
+  void smallBigint() {
```

Comment:
There is a good amount of copy-paste between smallBigint and largeBigint. The only differences are 'iterations' and 'batchSizes'. Perhaps, refactor using a single method: 

```
void bigint(int numIterations, const std::vector<vector_size_t>& batchSizes)
```